### PR TITLE
#21: Add description, about the team and demo video to the main page

### PR DESCRIPTION
**Pull request by rss-maksim**

### CODE REVIEW

**Review by IlyaKozak — APPROVED** [review]

File: src/app/main/pages/home-page/home-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    align-items: center;
+    flex-wrap: nowrap;
+
+    @media screen and (max-width: $tablet-breakpoint) {
+      flex-direction: column;
+    }
+  }
+
+  &__slogan {
+    font-size: 2.5rem;
```
[the remark this answers is not in the document]
Comment (rss-maksim) [reply src/app/main/pages/home-page/home-page.component.scss:20]: @hhharm Decided to use ```rem```

File: src/app/main/components/teammate-card/teammate-card.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<mat-card class="teammate-card">
```
[the remark this answers is not in the document]
Comment (rss-maksim) [reply src/app/main/components/teammate-card/teammate-card.component.html:1]: @hhharm Frankly speaking I didn't get it. Could you explain in more details.
Thanks.

File: src/app/main/components/about-team/about-team.component.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div class="about-team">
+  <h2 class="about-team__title">О команде</h2>
+  <div class="about-team__cards">
+    <app-teammate-card *ngFor="let mate of teammates$ | async" [mate]="mate"></app-teammate-card>
```
[the remark this answers is not in the document]
Comment (rss-maksim) [reply src/app/main/components/about-team/about-team.component.html]: Done

File: src/app/main/components/teammate-card/teammate-card.component.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    </mat-card-title>
+    <mat-card-subtitle>
+      <span>{{ mate.role }}</span>
+      <span class="teammate-card__location" *ngIf="mate.location">({{ mate.location }})</span>
+    </mat-card-subtitle>
+  </mat-card-header>
+  <mat-card-content class="teammate-card__content">
+    <ul class="teammate-card__contribution">
+      <h4 class="teammate-card__contribution-title">Вклад:</h4>
+      <li class="teammate-card__contribution-item" *ngFor="let item of mate.contribution">{{ item }}</li>
```
[the remark this answers is not in the document]
Comment (rss-maksim) [reply src/app/main/components/teammate-card/teammate-card.component.html]: Done

**Review by Ars6300 — APPROVED** [review]

**Review by hhharm — APPROVED** [review]

**Pull request merged**